registry_test.go

```diff
@@ -350,6 +350,67 @@ func TestConvert(t *testing.T) {
 				},
 			},
 		},
+		"one timer": {
+			setup: func(r *Registry) {
+				tmer := r.Timer("test.test", map[string]string{})
+				tmer.Record(1 * time.Second)
+			},
+			expectedOutputs: map[string]Metric{
+				"test.test": Metric{
+					Kind: "Timer",
+					Values: []TopValue{
+						TopValue{
+							Tags: []Tag{
+								Tag{Key: "statistic", Value: "count"},
+								Tag{Key: "nf.app", Value: "test"},
+								Tag{Key: "nf.cluster", Value: "test-main"},
+								Tag{Key: "nf.asg", Value: "test-main-v001"},
+								Tag{Key: "nf.region", Value: "us-west-1"},
+							},
+							Values: []*Value{
+								&Value{V: 1, T: 0},
+							},
+						},
+						TopValue{
+							Tags: []Tag{
+								Tag{Key: "statistic", Value: "totalTime"},
+								Tag{Key: "nf.app", Value: "test"},
+								Tag{Key: "nf.cluster", Value: "test-main"},
+								Tag{Key: "nf.asg", Value: "test-main-v001"},
+								Tag{Key: "nf.region", Value: "us-west-1"},
+							},
+							Values: []*Value{
+								&Value{V: 100000, T: 0},
+							},
+						},
+						TopValue{
+							Tags: []Tag{
+								Tag{Key: "statistic", Value: "totalOfSquares"},
+								Tag{Key: "nf.app", Value: "test"},
+								Tag{Key: "nf.cluster", Value: "test-main"},
+								Tag{Key: "nf.asg", Value: "test-main-v001"},
+								Tag{Key: "nf.region", Value: "us-west-1"},
+							},
+							Values: []*Value{
+								&Value{V: 100000, T: 0},
+							},
+						},
+						TopValue{
+							Tags: []Tag{
+								Tag{Key: "statistic", Value: "max"},
+								Tag{Key: "nf.app", Value: "test"},
+								Tag{Key: "nf.cluster", Value: "test-main"},
+								Tag{Key: "nf.asg", Value: "test-main-v001"},
+								Tag{Key: "nf.region", Value: "us-west-1"},
```

Comment:
Small, non-blocking suggestion - instead of repeat the same tags over and over, why not wrap them in a function that adds them to the tags you add on top? 

```
func withDefaultTags(tags ...Tag) {
	defaultTags := []Tag{...} //default tags
	for _, t := range tags {
		defaultTags := append(defaultTags, t)
	}
}
...
Tags: withDefaultTags(Tag{Key: "foo", Value: "bar"}, Tag...)
```